src/Raven.Server/Documents/Handlers/DocumentHandler.cs

```diff
@@ -528,5 +531,38 @@ public void Dispose()
         {
             _document?.Dispose();
         }
+
+        public DynamicJsonValue Serialize()
+        {
+            var ret = new DynamicJsonValue
+            {
+                [IdKey] = _id,
+                [DocumentKey] = _document,
+                [ChangeVectorKey] = _expectedChangeVector
+            };
+
+            return ret;
+        }
+
+        public static MergedPutCommand Deserialize(BlittableJsonReaderObject mergedCmdReader, DocumentDatabase database, JsonOperationContext context)
+        {
+            if (mergedCmdReader.TryGet(IdKey, out string id) == false)
+            {
+                throw new Exception($"Cant read {IdKey} from {nameof(MergedPutCommand)}");
```

Comment:
`Cant` -> `Can't` (and elsewhere as well)

Reply:
This hasn't been fixed

Reply:
Don't throw `Exception`

Reply:
done